src/status_im2/contexts/wallet/address_watch/view.cljs

```diff
@@ -37,12 +39,18 @@
           :container-style {:margin-right 12
                             :flex         1}
           :weight          :monospace
-          :on-change       #(reset! input-value %)
+          :on-change       #(reset! input-value (.. % -nativeEvent -text))
           :default-value   @input-value}]
         [quo/button
          {:icon-only? true
           :type       :outline} :i/scan]]
        [rn/view {:style (style/button-container bottom)}
-        [quo/text "[WIP] Bottom Actions"]]])))
+        [quo/bottom-actions
```

Comment:
btw, here we need `customization-color` on the button too. on this page it's the users custom color 👍 we have this data available on many pages. 
you can get it by the following sub `{:keys [customization-color]} (rf/sub [:profile/multiaccount])`

Reply:
great suggestion! i also used this customization color on the second screen, so the user won't need to touch color picked if he would like to preserve the old color